Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/core/ErrorConstants.java

```diff
@@ -0,0 +1,8 @@
+package edu.asu.conceptpower.servlet.core;
+
+public interface ErrorConstants {
+
+    public final static String INDEXERSTATUS = "IndexerStatus";
+    public final static String INDEXER_RUNNING = "Indexer running. Please try after sometime";
```

Comment:
this is not a constant but an error message. This should be in a message file. Also, it should say: "The indexer is currently running. Please try again in a minute."